notebooks/monitoring.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+
+declare -a TEMP=$(mktemp /temp_monitoring.XXXXXXXX)
+
+
+function get_disk_info() {
+        # df command and cromwell root field
+        disk_info=$(df | grep cromwell_root)
+        echo "$disk_info"
+}
+
+
+function get_disk_usage() {
+        # get disk usage field
+        echo $(get_disk_info | awk '{ print $5 }')
+}
+
+
+function get_mem_info() {
+        # /proc/meminfo
+        mem_info=$(cat /proc/meminfo)
+        echo "$mem_info"
+}
+
+
+function get_mem_available() {
+        # mem unused from /proc/meminfo
+        mem_available=$(get_mem_info | grep MemAvailable | awk 'BEGIN { FS=" " } ; { print $2 }')
+        echo "$mem_available"
+}
+
+
+function get_mem_total() {
+        # mem total from /proc/meminfo
+        mem_total=$(get_mem_info | grep MemTotal | awk 'BEGIN { FS=" " } ; { print $2 }')
+        echo "$mem_total"
+}
+
+
+function get_mem_usage() {
+        # memTotal and memAvailable
+        mem_total=$(get_mem_total)
```

Comment:
For local vars, it's good practice to declare them with `local -r`. i.e.
```
local -r mem_total=$(get_mem_total)
```
The `local` keeps them from getting pushed into global scope, and `-r` marks them as read-only.